src/frontend/context/AppContext.test.tsx

```diff
@@ -41,3 +60,51 @@ describe('erStegUtfyltFraFør', () => {
         expect(hookResult.current.erStegUtfyltFrafør(nåværendeStegIndex)).toEqual(false);
     });
 });
+
+describe('nullstillSøknadsObject', () => {
+    test('Skal nullstille søknadsobjekt bortsett fra person hentet fra backend og søknadstype', () => {
+        const wrapper = ({ children }) => <AppProvider>{children}</AppProvider>;
+        const { result } = renderHook(() => useApp(), { wrapper });
+
+        const søknadHalvveisUtfylt: ISøknad = {
+            ...søknadEtterRespons,
+            erNoenAvBarnaFosterbarn: {
+                id: OmBarnaDineSpørsmålId.erNoenAvBarnaFosterbarn,
+                svar: ESvar.JA,
+            },
+            oppholderBarnSegIInstitusjon: {
+                id: OmBarnaDineSpørsmålId.oppholderBarnSegIInstitusjon,
+                svar: ESvar.JA,
+            },
+            erBarnAdoptertFraUtland: {
+                id: OmBarnaDineSpørsmålId.erBarnAdoptertFraUtland,
+                svar: ESvar.JA,
+            },
+            oppholderBarnSegIUtland: {
+                id: OmBarnaDineSpørsmålId.oppholderBarnSegIUtland,
+                svar: ESvar.JA,
+            },
+        };
+
+        act(() => {
+            result.current.settSøknad(søknadHalvveisUtfylt);
+        });
+        expect(result.current.søknad).toEqual(søknadHalvveisUtfylt);
+        act(() => {
+            result.current.nullstillSøknadsobjekt();
+        });
+        expect(result.current.søknad).toEqual(søknadEtterRespons);
+    });
+});
+
+describe('avbrytSøknad', () => {
+    test('Ved avbryt skal sisteUtfylteStegIndex settes til -1', () => {
+        const wrapper = ({ children }) => <AppProvider>{children}</AppProvider>;
+        const { result } = renderHook(() => useApp(), { wrapper });
+        result.current.sisteUtfylteStegIndex = 3;
```

Comment:
Kan gjerne deconstructe result.current her slik at det blir litt enklere å lese resten

Reply:
Det går ikke :( da får den en gammel current. Så må hente ut current akkurat når den skal brukes for å få den oppdaterte current.